pkg/server/server.go

```diff
@@ -58,15 +58,30 @@ var (
 func NewServer() (server *Server) {
 	router := mux.NewRouter()
 
+	// Top Level
 	router.Path("/environments").Methods("POST").HandlerFunc(createEnvironment)
+
+	// Org:Env Level
 	router.Path("/environments/{org}:{env}").Methods("GET").HandlerFunc(getEnvironment)
 	router.Path("/environments/{org}:{env}").Methods("PATCH").HandlerFunc(updateEnvironment)
 	router.Path("/environments/{org}:{env}").Methods("DELETE").HandlerFunc(deleteEnvironment)
 	router.Path("/environments/{org}:{env}/deployments").Methods("POST").HandlerFunc(createDeployment)
 	router.Path("/environments/{org}:{env}/deployments").Methods("GET").HandlerFunc(getDeployments)
+
+	router.Path("/environments/{org}:{env}/config").Methods("GET").HandlerFunc(getConfigs)
+	router.Path("/environments/{org}:{env}/config").Methods("POST").HandlerFunc(createConfig)
+
+	// Deployment Level
 	router.Path("/environments/{org}:{env}/deployments/{deployment}").Methods("GET").HandlerFunc(getDeployment)
 	router.Path("/environments/{org}:{env}/deployments/{deployment}").Methods("PATCH").HandlerFunc(updateDeployment)
 	router.Path("/environments/{org}:{env}/deployments/{deployment}").Methods("DELETE").HandlerFunc(deleteDeployment)
+
+	// Config Level
```

Comment:
I'm saying instead of having to make a single call per key/value pair, allowing the user to bulk add/update key/value pair(s).